#!/usr/bin/python3
def raise_exception_msg(message=""):
    try:
        print(CetteVariableNexistePas)
    except NameError:
        print(message)
